### ISSUE TITLE
NTR: CT or MRI: nasal mass (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8983]
New term request CT or MRI: nasal mass (elthat_140106053829):   UMLS: 0746765

**Comment by pnrobinson** [comment]
@MickeySegal
is this referring to a mass within the nasal cavity?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000997
- label: Intranasal mass
- parents: [Abnormal nasal cavity morphology (HP:0010640)](https://hpo.jax.org/app/browse/term/HP:0010640)
- pmid: 
- def: An abnormal enlargement or swelling localized to the nasal cavity. The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8983
- synonyms:
